gfeloqua.class.php

```diff
@@ -1682,13 +1685,17 @@ function retry_failed_submissions(){
 	 * @return array        Entries
 	 */
 	function get_failed_entries( $forms = array() ){
+
+		if ( empty( $forms ) ) {
+			return;
```

Comment:
I think if we return an empty array here, that would keep from having to check if the returned value is an array. The expected response of this function should be an array.